src/shapes/Ellipse.ts

```diff
@@ -22,6 +27,15 @@ export class Ellipse extends FabricObject {
 
   static cacheProperties = [...cacheProperties, 'rx', 'ry'];
 
+  static ownDefaults: Record<string, any> = ellipseDefaultValues;
+
+  getDefaultValues() {
+    return {
+      ...super.getDefaultValues(),
+      ...Ellipse.ownDefaults,
+    };
```

Comment:
if we are spreading and abandoning the shared object concept, is there a need for `ownDefaults`?

Reply:
yes, because is how you change the defaults.
Otherwise how do you change them?

Also in the description i was explaining that emptying the ownDefaults is a clean way to go back to defaults on the prototype if the developer prefer

Reply:
`Object.getDefaultValues=()=> ({})`